sanitizers/app/build.gradle

```diff
@@ -37,6 +37,12 @@ android {
         asan {
             initWith debug
             debuggable true
+            compileSdk 27
```

Comment:
compileSdk is already 32 above, and you really want to change minSdk. But it's probably fine to just leave this out.

Reply:
Solved it. Thanks!